data/mods/Xedra_Evolved/mutations/mutations.json

```diff
@@ -445,6 +466,112 @@
     "anger_relations": [ [ "VAMPIRE", -95 ], [ "RENFIELD", -95 ] ],
     "enchantments": [ { "values": [ { "value": "STAMINA_REGEN_MOD", "add": 0.1 }, { "value": "METABOLISM", "multiply": 0.5 } ] } ]
   },
+  {
+    "type": "mutation",
+    "id": "TRUE_VAMPIRE_POWER",
+    "points": 0,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "name": { "str": "True Vampire's Potence" },
+    "description": "Your new status as a true vampire has empowered your other abilities.",
+    "enchantments": [ "true_vampire_power_real" ]
+  },
+  {
+    "type": "enchantment",
+    "id": "true_vampire_power_real",
+    "condition": "ALWAYS",
+    "mutations": [ "TRUE_VAMPIRE_POWER_1", "TRUE_VAMPIRE_POWER_2", "TRUE_VAMPIRE_POWER_3", "TRUE_VAMPIRE_POWER_4", "TRUE_VAMPIRE_POWER_5" ]
+  },
+  {
+    "type": "mutation",
+    "id": "TRUE_VAMPIRE_POWER_1",
+    "points": 0,
+    "player_display": false,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "name": { "str": "True Vampire's Potence 1" },
+    "description": "Fake mutation to boost vampiric abilities."
+  },
+  {
+    "type": "mutation",
+    "id": "TRUE_VAMPIRE_POWER_2",
+    "points": 0,
+    "player_display": false,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "name": { "str": "True Vampire's Potence 2" },
+    "description": "Fake mutation to boost vampiric abilities."
+  },
+  {
+    "type": "mutation",
+    "id": "TRUE_VAMPIRE_POWER_3",
+    "points": 0,
+    "player_display": false,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "name": { "str": "True Vampire's Potence 3" },
+    "description": "Fake mutation to boost vampiric abilities."
+  },
+  {
+    "type": "mutation",
+    "id": "TRUE_VAMPIRE_POWER_4",
+    "points": 0,
+    "player_display": false,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "name": { "str": "True Vampire's Potence 4" },
+    "description": "Fake mutation to boost vampiric abilities."
+  },
+  {
+    "type": "mutation",
+    "id": "TRUE_VAMPIRE_POWER_5",
+    "points": 0,
+    "player_display": false,
+    "starting_trait": false,
+    "purifiable": false,
+    "valid": false,
+    "name": { "str": "True Vampire's Potence 5" },
+    "description": "Fake mutation to boost vampiric abilities."
+  },
```

Comment:
If you're planning on inserting these into the Jmath, you don't need 5 separate mutations to do it. You can just do 

> {
>     "type": "jmath_function",
>     "id": "vampire_total_tier_four_traits",
>     "num_args": 0,
>     "return": "(u_has_trait('TRUE_VAMPIRE_POWER') * 5)"
>   }

...and True Vampire's Potence will count as five mutations' worth for the scaling math.

Reply:
Good point. I'll begin setting that up in a soon-ish push